### PR TITLE
Add color utils actions

### PR DESCRIPTION
**Pull request by lolodomo**
Related to openhab/openhab-core#3749

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: addons/actions.md [hunk 1]
```diff
@@ -202,6 +202,19 @@ if ((thingStatusInfo !== null) && (thingStatusInfo.getStatus().toString() == "ON
...
+- `double[] hsbToXY(HSBType hsb)`: transforms HSB to CIE 1931 "xy" format using default color Gamut, returning an array of three double with the closest matching CIE 1931 x,y,Y in the range 0.0000 to 1.0000
+- `double[] hsbToXY(HSBType hsb, double[] gamutR, double[] gamutG, double[] gamutB)`: transforms HSB to CIE 1931 "xy" format using specific color Gamut, returning an array of three double with the closest matching CIE 1931 x,y,Y in the range 0.0000 to 1.0000
+- `HSBType rgbToHsb(int[] rgb)`: transforms RGB (array of three or four int with the RGB(W) values in the range 0 to 255) to HSB
+- `HSBType rgbToHsb(PercentType[] rgb)`: transforms RGB (array of three PercentType with the RGB values in the range 0 to 100 percent) to HSB
+- `HSBType xyToHsb(double[] xy)`: transforms CIE 1931 "xy" format (array of double with CIE 1931 x,y[,Y] in the range 0.0000 to 1.0000 with Y being optional) to HSB using default color Gamut
+- `HSBType xyToHsb(double[] xy, double[] gamutR, double[] gamutG, double[] gamutB)`: transforms CIE 1931 "xy" format (array of double with CIE 1931 x,y[,Y] in the range 0.0000 to 1.0000 with Y being optional) to HSB using specific color Gamut
+
+
```
Comment (stefan-hoehn) [inline addons/actions.md:217]: Can you please delete that one line?

Thanks for writing that down, Laurent, very much appreciated!

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]

File: addons/actions.md (hunk 1 above)
Comment (lolodomo) [reply addons/actions.md:217]: What line number exactly ?
Do you mean one of the blank lines ? Line 216 ?

Comment (stefan-hoehn) [reply addons/actions.md:217]: 216 or 217, yes. There should only be one by our definitions.

Comment (lolodomo) [reply addons/actions.md]: Done

**Review by stefan-hoehn — APPROVED** [review]
Thx! Unfortunately there is another file you did not change that is breaking the build.

**Comment by lolodomo** [comment]
@stefan-hoehn : if you merge a fix for synonology.md, then I will merge the updated main branch into all my branches and the problem will be solved. Very easy. So if you know what has to be fixed in synonology.md, please do it and merge it.

**Comment by stefan-hoehn** [comment]
> @stefan-hoehn : if you merge a fix for synonology.md, then I will merge the updated main branch into all my branches and the problem will be solved. Very easy. So if you know what has to be fixed in synonology.md, please do it and merge it.

Ok, fine, I have discovered even more in a different PR, so that's be me task today to verify all other places as well. I'll fix it and then rerun the build here.

**Comment by lolodomo** [comment]
> Ok, fine, I have discovered even more in a different PR, so that's be me task today to verify all other places as well. I'll fix it and then rerun the build here.

I just submitted a PR fixing the problems in synology.md.
Please check and merge it.

**Comment by stefan-hoehn** [comment]
@lolodomo I fixed that one issue #2160 and reran the build here but it still fails. 

Could it be that your PR needs to be rebased? I could do 
image
but then it would be automatically merged without the build being run.

**Comment by lolodomo** [comment]
Yes, I will rebase. I do it in half an hour.

**Comment by lolodomo** [comment]
All my PRs are fine now after a rebase. You will be able to merge them ;)

**Pull request merged**